ts-select2.js

```diff
@@ -185,6 +185,25 @@
                 getInput().on('blur.select2', function() {
                     element.trigger('blur');
                 });
+
+                // Watch for the model change
+                scope.$watch(function() {
```

Comment:
Ezt a kódot meg lehet valósítani a saját direktívánkban is, sőt onnan származik az alapja, viszont ki kellett egészítenem, hogy akkor is ürítse a select-et, ha null-ra állítjuk a model értékét, valamint nem tudtam új értéket kiválasztatni programkódból. És mivel ez a tulajdonság ebből a direktívából is hiányzik arra gondoltam orvosolhatnánk akár itt is, márha úgyis saját fork-unk van. Ami maradt a formly-s nslSelectWatcher direktívában az a multiselectes újbóli kiválasztás inicializáláskor, amit nem látom, hogy kell-e egyáltalán, vagy azt is át kellene/lehetne mozgatni ide.